docs/conf.py

```diff
@@ -1,15 +1,16 @@
 import codecs
-import os
 import re
 
+from pathlib import Path
+
 
 def read(*parts):
     """
     Build an absolute path from *parts* and and return the contents of the
     resulting file.  Assume UTF-8 encoding.
     """
-    here = os.path.abspath(os.path.dirname(__file__))
-    with codecs.open(os.path.join(here, *parts), "rb", "utf-8") as f:
+    here = Path(__file__).parent.absolute()
+    with codecs.open(str(here.joinpath(*parts)), "rb", "utf-8") as f:
```

Comment:
```suggestion
    HERE = Path(__file__).parent.absolute()
    with HERE.joinpath(*parts).open("r", encoding="utf-8") as f:
```
No need to convert to a `str`, almost every open function/method accepts *path-like* objects. Don't open a file in binary when you apply an encoding.

Other parts use `HERE`.